apps/backend/package.json

```diff
@@ -13,7 +13,9 @@
   "license": "ISC",
   "dependencies": {
     "@repo/db": "*",
+    "@types/cookie-parser": "^1.4.7",
```

Comment:
these should be dev dependencies right?

Reply:
yes, updated and moved it from dep to devDep